docs/guidelines/postman.md

```diff
@@ -0,0 +1,122 @@
+# Postman
+
+!!! info
+    Postman is an API platform for building and using APIs. Postman simplifies each step of the API lifecycle and streamlines collaboration so you can create better APIs—faster.
+
+## Changes in 2023
+
+During 2023, Postman [announced a change 🔗](https://blog.postman.com/announcing-new-lightweight-postman-api-client/) on how the solution works by making it mainly "cloud only", meaning they expect any users of the solution to always have an account and being logged in. When utilizing Postman, it's crucial to note that being logged in will _automatically_ sync data to the cloud. As a result, users must exercise caution regarding the nature of data being transmitted. It's especially important to pay attention to the **data's classification**, both within the **requests** (such as API keys and personal information) and in the **responses**.
+
+## Benefits of being logged in with an user account
+Logging in with a user account enables most of the features within Postman, including:
+
+- Workspaces
+- Collections
+- Sharing and collaboration
+- **Automatic synchronization to the cloud / backup**
+
+However, the automatic synchronization to the cloud imposes both security and regulationary concerns. Here are the results of a _a simple_ Threat Model exercise:
+
+| Threat | Description | Mitigation |
+|---|---|---|
+|Information disclosure|The API being tested has sensitive data in the responses - could be `confidential` or `personal` data - which would automatically be uploaded to the cloud|Know what data classification the data in the **requests and responses** belongs to and if uncertain, try to avoid using Postman. When using automatic testing, ensure tests are being performed against a "synthetic" test environment|
+|Account takeover / session hijacking|The account is breached and malicious actors can log in to Postman using your account|Only use SSO as there are protective and preventive measures in place to avoid/detect malicious actors logging in|
+|Data breach at Postman servers|A malicious actor has obtained data residing on the Postman servers|Avoid storing sensitive data that is being synchronized towards Postman servers|
+|Credentials stored unencrypted|Storing credentials in plain-text fields, in local plain-text files|Use proper solutions when handling credentials, like Postman Vault or use OAuth 2.0 authorization flows without client secrets (require user to log in through SSO)|
+
+
+## Best practices
+The automatic synchronization to the cloud imposes both security and regulationary concerns.
+Following the best practices will mitigate some of those concerns.
+
+### SSO
+_Never_ use a private or non-Equinor account in an Equinor context.
+If needing the full feature set of Postman, request access through AccessIT and use SSO for login. 
```

Comment:
Should we add a link to the "Enterprise log in" - https://identity.getpostman.com/enterprise/login to make things very explicit?

Reply:
Yes - that will be helpful for those who use the browser to login to Postman. For those using the Desktop Client the login flow will be through that.